src/routes/books.js

```diff
@@ -3,10 +3,20 @@
 const express = require('express');
 const router = express.Router();
 
+const auth = require('../auth/middleware.js');
+router.use(auth);
+
 router.get('/books', handleGetAll);
```

Comment:
You added your auth middleware but you need to pass it through your route handlers!